lib/vcr/errors.rb

```diff
@@ -284,14 +292,22 @@ def record_mode_suggestion
         record_modes = current_cassettes.map(&:record_mode)
 
         if record_modes.all?{|r| r == :none }
-          [:deal_with_none]
+          none_suggestion
         elsif record_modes.all?{|r| r == :once }
           [:delete_cassette_for_once]
         else
           []
         end
       end
 
+      def none_suggestion
+        if current_cassettes.any? {|c| !File.exist?(c.file) }
+          [:none_without_file]
+        else
+          [:deal_with_none]
```

Comment:
Should we add something for this scenario when a file does exist?

Running the suite locally and checking the coverage, this line is unreachable now.

<img width="522" alt="image" src="https://github.com/user-attachments/assets/80c0ed37-abc8-4ba0-82ba-fb7494ec6598">

Reply:
Added tests for this, nice catch!